plugins/juju/04known_bugs

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import subprocess
+
+from common import helpers, constants, searchtools
+from common.known_bugs_utils import add_known_bug
+from juju_common import (
+    JUJU_LOG_PATH
+)
+
+
+def detect_bug1910958():
+    """Unit fails to start complaining there are members in the relation."""
+    result = {
+        1910958: {
+            "description": ("Unit fails to start complaining there are "
+                            "members in the relation."),
+            "match": ""
+            }
+        }
+    pattern = (
+        r'.* manifold worker returned unexpected error: failed to initialize '
+        'uniter for "[A-Za-z0-9\\-]+": cannot create relation state tracker: '
+        'cannot remove persisted state, relation \\d+ has members'
+    )
+
+    try:
+        s = searchtools.FileSearcher()
+        s.add_search_term(pattern, [0], f"{JUJU_LOG_PATH}/*.log")
+
+        results = s.search()
+
+        disp_text = []
+        for fpath in results.files:
+            for r in results.find_by_path(fpath):
+                line = f"{r.source}:{r.line}"
+                if len(line) > constants.MAX_MATCH_CHARS:
+                    disp_text.append(line[:constants.MAX_MATCH_CHARS] + "...")
+                else:
+                    disp_text.append(line.strip("\n"))
+
+        result[1910958]["match"] = "\n".join(disp_text)
+        add_known_bug(1910958)
+
+    except subprocess.CalledProcessError:
+        return None
+
+    return result
+
+
+KNOWN_ISSUES = [detect_bug1910958]
+
+
+def main():
+    issues_found = {"launchpad": []}
```

Comment:
This code can be simplified now since the indentation handling is all done in the plugin_yaml code. See other plugins for an example but basically all you need to do is something like:

if issues_found:
        helpers.HOTSOSYaml.dump(issues_found)

and ignore all the indent checking stuff.